README.md

```diff
@@ -57,36 +51,47 @@ Package Name | Minimum Version
 
 Package Name | Minimum Version
 ------------ |:--------------:
-[sphinx](http://www.sphinx-doc.org/) | 1.4
-[inkscape](http://inkscape.org) |
+[`sphinx`](http://www.sphinx-doc.org/) | 1.8
 
-> **Note:** We recommend using [pip](https://pypi.org/project/pip) to manage Python modules.
-
-Python Module |
----------- |
-sphinx_rtd_theme |
-recommonmark |
+Python Module | Required version
+------------- |:---------------:
+`sphinx_rtd_theme` | 0.4.3
+`recommonmark` | 0.5.0
+`docutils` | 0.12
 
 
 ### Build process ###
 
-Gaffer also depends on a number of 3rd-party libraries and python modules, many of which are not entirely straightforward to build. We therefore recommend using the latest pre-built dependencies from the [Gaffer dependencies project](https://github.com/GafferHQ/dependencies/releases). These are used in our automated test builds and so are guaranteed to be up-to-date with Gaffer's requirements.
+```bash
+git clone https://github.com/GafferHQ/gaffer.git
+cd gaffer
+```
+
+Gaffer depends on a number of 3rd-party libraries. We recommend using the latest pre-built dependencies from the [Gaffer dependencies project](https://github.com/GafferHQ/dependencies/releases). These are used in our automated test builds and so are guaranteed to be up-to-date with Gaffer's requirements.
 
-Once you've downloaded the dependencies, unpack them and move/rename them to the directory in which you want to make your Gaffer build. We'll refer to this location as `<BUILD_DIR>` in the instructions below. Before continuing, make sure the dependencies are unpacked as `<BUILD_DIR>/include`, `<BUILD_DIR>/lib`, etc.
+The dependencies distribution forms the basis of a Gaffer build and the root of the expanded archive becomes the `BUILD_DIRECTORY` we pass to `scons`.
```

Comment:
The dependencies distribution forms
↓
The latest release of dependencies forms

Reply:
It's not always the latest release. If you're building `0.55_maintenance`, then its a previous version.